### ISSUE TITLE
Fails to run on RHEL7

### ISSUE THREAD

**Issue by kohlivarun5** [rescript-lang/rescript-vscode#596]
With a recent version, the extension crashes and doesn't work on RHEL7 with the following errors:

```
Error: Command failed: /root/.vscode-server/extensions/chenglou92.rescript-vscode-1.6.0/server/analysis_binaries/linux/rescript-editor-analysis.exe diagnosticSyntax /tmp/rescript_format_file_21683_0.res
/root/.vscode-server/extensions/chenglou92.rescript-vscode-1.6.0/server/analysis_binaries/linux/rescript-editor-analysis.exe: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /root/.vscode-server/extensions/chenglou92.rescript-vscode-1.6.0/server/analysis_binaries/linux/rescript-editor-analysis.exe)
```

What is the suggested workflow for usage of the latest version on RHEL7 ? The version I have installed that fails is `v1.6.0`

**Comment by kohlivarun5** [comment]
Resurfacing this, I see it also fails on RHEL8 : 

> /root/.local/share/code-server/extensions/chenglou92.rescript-vscode-1.50.0-universal/server/analysis_binaries/linux/rescript-editor-analysis.exe: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /root/.local/share/code-server/extensions/chenglou92.rescript-vscode-1.50.0-universal/server/analysis_binaries/linux/rescript-editor-analysis.exe)

**Comment by cknitt** [comment]
Fixed in #976 by statically linking the Linux binaries.

**Comment by kohlivarun5** [comment]
Thanks for the fix. Can you please make a release and publish ?
